apps/fuzzyclock/fuzzy_clock.star

```diff
@@ -117,9 +246,9 @@ def get_schema():
             ),
             schema.Dropdown(
                 id = "dialect",
-                name = "Dialect",
+                name = "Language",
                 icon = "language",
-                desc = "British or American English",
+                desc = "Language in which to display time",
                 default = dialectOptions[0].value,
```

Comment:
For now, I think it would be good to default to `en-US`, as that's likely the language of the bulk of the users. At some point in the future I hope we can provide better localization primitives.

Reply:
Oh, absolutely - this was an oversight. Fixed.